src/model/dfcxx/include/dfcxx/typedefs.h

```diff
@@ -61,7 +61,9 @@ enum Scheduler {
 
 typedef std::unordered_map<dfcxx::Ops, unsigned> DFLatencyConfig;
 
-// Macro substitutions for indexes in a std::vector for paths.
+// Has to be incremented with every new output format.
+#define OUTPUT_FORMATS_COUNT 1

```

Comment:
This solution is bad and potentially leads to errors. Every developer should keep in mind, that the number of supported output formats should be explicitly specified.

Reply:
Tried to fix it using a `enum` with a utility value `COUNT`. I have already thought of this initially, but wanted to avoid enum->integer cast issues, using `#define`s instead of enumerations.